app/src/main/java/com/belatrixsf/allstars/ui/common/views/KeywordView.java

```diff
@@ -0,0 +1,107 @@
+package com.belatrixsf.allstars.ui.common.views;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.text.TextUtils;
+import android.util.AttributeSet;
+import android.view.LayoutInflater;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.belatrixsf.allstars.R;
+
+/**
+ * Created by icerrate on 06/05/2016.
+ */
+public class KeywordView extends LinearLayout {
+
+    private TextView keyboardTextView;
+    private String value;
+
+    public KeywordView(Context context) {
+        super(context);
+        initView();
+    }
+
+    public KeywordView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+        initView();
+        TypedArray typedArray = context.obtainStyledAttributes(attrs, R.styleable.KeywordView);
+
+        String hintText = typedArray.getString(R.styleable.KeywordView_key_text);
+        keyboardTextView.setText(hintText);
+
+        typedArray.recycle();
+    }
+
+    private void initView() {
+        LayoutInflater inflater = (LayoutInflater) getContext().getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+        inflater.inflate(R.layout.view_keyword, this, true);
+        keyboardTextView = (TextView) findViewById(R.id.keyword);
+    }
+
+    public void setData(String data) {
+        if (data != null && !data.isEmpty()) {
+            keyboardTextView.setText(data);
+            value = data;
```

Comment:
do we really need to have a global reference for this string? or later when saving we could get the text from the TextView.gettext()?